presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -59,6 +59,46 @@ General Aggregate Functions
 
     Returns the geometric mean of all input values.
 
+.. function:: grouping(col1, ..., colN) -> integer
+
+    Returns a bit set converted to decimal, indicating which columns are present in a grouping. The function
+    must be used in conjunction with ``GROUPING SETS``, ``ROLLUP`` or ``CUBE`` and its arguments must
+    match exactly the columns referenced in the corresponding ``GROUPING SETS``, ``ROLLUP`` or ``CUBE``.
+
+    To compute the resulting bit set for a particular row, bits are assigned to the argument columns with
+    the rightmost column being the least significant bit. For a given grouping, a bit is set to 0 if the
```

Comment:
Ah, I see. Yeah, the key is the phrase "aggregating over" -- it doesn't mean grouped on, it means that the column was used as an argument to aggregation functions.

This is consistent with another note in the spec:

> NOTE 209 — Thus the value of a `<grouping operation>` is 0 (zero) if the grouping column referenced by the `<grouping operation>` is among the `GCRi,j` and 1 (one) if it is not.